src/wazuh_modules/wm_vulnerability_scanner.c

```diff
@@ -68,6 +68,10 @@ void* wm_vulnerability_scanner_main(wm_vulnerability_scanner_t * data)
         {
             cJSON_AddStringToObject(data->vulnerability_detection, "feed-update-interval", "60m");
         }
+        if (!cJSON_GetObjectItem(data->vulnerability_detection, "clusterName"))
+        {
+            cJSON_AddStringToObject(data->vulnerability_detection, "clusterName", get_cluster_name());
```

Comment:
The call to `get_cluster_name()` here is leaking memory because cJSON creates another copy of the string.

Reply:
Done